blog/_data/authors.yml

```diff
@@ -59,3 +59,15 @@ Coco Wang:
     - label: "LinkedIn"
       icon: "fab fa-fw fa-linkedin"
       url: "https://www.linkedin.com/in/coco-wang-2399041b6/"
+
+Saurabh Sharma:
+  name        : "Saurabh Sharma"
+  bio         : "Product Manager for Cloud Native Storage initiatives"
+  avatar      : "" # Replace this with your avatar image
```

Comment:
You need to add an avatar here so that your picture appears above your name in the post. Look at other authors/posts for examples.

Reply:
This is fixed